src/model/advertisement.ts

```diff
@@ -31,7 +33,13 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'mostpop',
```

Comment:
I noticed that [this AdSlot.scala file](https://github.com/guardian/frontend/blob/04a2b9112c5bf792510965c405cbd406b083288e/common/app/common/commercial/AdSlot.scala) has different values in it - how do we know which ones are right?

Reply:
Going to ask Ioanna